database_standard.qmd

```diff
@@ -16,5 +15,63 @@ library(traits.build)
 my_kable_styling <- util_kable_styling_html
 ```
 
-XXX Describe the emerhign traits.build data standard
+### Overview
+
+A single universally accepted trait database standard does not currently exist among plant ecologists, limiting our ability to merge together distinct databases.\ 
```

Comment:
Why end with `\`

Reply:
Forced line break - I can't figure out why some are immediately reformatted and others remain in the text.